docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -80,7 +80,8 @@ This property is optional; the default is ``_schema``.
 .. note::
 
     Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
-    However, not all the fields are always detected and you might need to update the schema manually or using ``ALTER TABLE``.
+    However, not all the fields are always detected and you might need to update the schema directly on the MongoDB
```

Comment:
There are multiple issues in the passage. Suggest:

However, not all the fields are always detected. In this case, update the schema directly on the MongoDB
    entry, or through Trino using ``ALTER TABLE``."

The ALTER TABLE solution needs an example, as its use here is non-intuitive. I'd be specific with an example field that is missing, and how to construct the statement specifically to contend with that.